content/blog/getting-started-with-hpe-greenlake-api-for-backup-and-recovery.md

```diff
@@ -0,0 +1,591 @@
+
+![API Request body JSON to compose protection Store](/img/api-to-create-protection-stores-request-json-body.png)
+
+4. After roughly about 5 minutes, the cloud protection store was completely created based on response of the following `GET /data-services/async-operations` [API](https://developer.greenlake.hpe.com/docs/greenlake/services/data-services/public/openapi/data-services-public-v1beta1/operation/GetAsyncOperation/) as shown in the below figure. 
+
+> ***Note:*** that I used a set of selection parameters in below figure to summarize the information from this task information:
+>
+> ```shellsession
+> GET /data-services/v1beta1/async-operations/{{taskId}}/select=associatedResources,createdAt,displayName,customerId,logMessages,progressPercent,state
+> ```
+>
+> I copied the task’s id from the response header’s location value of the prior API execution into a Postman’s variable called `{{taskId}}`, and incorporated `{{taskId}}` variable to the `async-operations` API execution.
+
+![Task completion on POST protection-stores](/img/api-async-on-post-protection-stores.png)
+
+5. For reference, I went into the Protection Stores menu as shown in the below figure to show that HPE GreenLake Backup and Recovery, and confirmed that the desired protection store was available.
+
+![Validating cloud protection store has been created](/img/ui-validated-cloud-protection-store-completed.png)
+
+## Creating a Protection-Policy
+
+One of the common use-case that every user of HPE GreenLake Backup and Recovery will deploy is the creation of a protection-policy. This resource is important because it sets up the schedule for creation of recovery point; additionally, it sets up the flow of a recovery point from a primary storage to a storage snapshot, to on-premises protection-store and eventually to the cloud protection-store. 
+
+A protection policy contain several JSON objects that are displayed in the below figure.
+
+1. A Schedule that contains the frequency for creation of the copy and time to retain the copies.
+2. Protection store where the copies are stored: snapshot (in the primary array), on-premises store, and cloud store.
+3. Length of time when the copy is being locked to satisfy immutability.
+4. The pre-script information contains the link to the scripts to be executed prior to the creation of the copy.
+5. The post-script information contains the link to the scripts to be executed after the creation of the copy.
+
+![API request body JSON for protection policy creation](/img/api-body-json-request-for-protection-policies.png)
+
+﻿*The above figure shows the guide for protection-policies request body JSON structure*
+
+To simplify this example, I created a three-tier protection-policy for VMware as depicted by this snippet from the protection policy’s menu.
+
+![3 Tiers protection policy ](/img/backup-protection-policy-with-3-tiers.png)
+
+The list of the steps to create this protection policy:
```

Comment:
Modify to read "**This is** the list of steps **used** to create this protection policy:"